micro-ui/web/micro-ui-internals/packages/modules/core/CHANGELOG.md

```
@@ -1,5 +1,8 @@
 # Changelog 
 
+## [1.8.36]  [27-May-2025]
+- FEATURE/CCSD-216 : Privacy component alignment changes
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)